### PR TITLE
Remove postgresql content for 2.0.0

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Remove postgresql content for 2.0.0

##### ISSUE TYPE

- Removal Pull Request

##### COMPONENT NAME
postgresql content

### CODE REVIEW

File: meta/runtime.yml [hunk 1]
```diff
@@ -371,6 +371,50 @@ plugin_routing:
...
+    postgresql_set:
+      redirect: community.postgresql.postgresql_set
+    postgresql_slot:
+      redirect: community.postgresql.postgresql_slot
+    postgresql_subscription:
+      redirect: community.postgresql.postgresql_subscription
+    postgresql_table:
+      redirect: community.postgresql.postgresql_table
+    postgresql_tablespace:
+      redirect: community.postgresql.postgresql_tablespace:
```
Comment (felixfontein) [inline meta/runtime.yml:413]:
```suggestion
      redirect: community.postgresql.postgresql_tablespace
```

**Comment by felixfontein** [comment]
Todo:
- [ ] fix YAML error in meta/runtime.yml (see suggestion above)
- [ ] rebase
- [ ] remove `postgresql_*` symlinks from `plugins/modules/`

**Review by felixfontein — APPROVED** [review]
LGTM, assuming odbc tests still pass :)

**Review by felixfontein — APPROVED** [review]

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@Andersson007 thanks a lot!

**Comment by Andersson007** [comment]
@felixfontein thanks very much for reviewing!

**Pull request merged**